src/fast_graph_builder.rs

```diff
@@ -242,18 +271,63 @@ impl FastGraphBuilder {
 pub struct Params {
     pub hierarchy_depth_factor: f32,
     pub edge_quotient_factor: f32,
+    /// The maximum number of settled nodes per witness search performed when priorities are
+    /// calculated for all nodes initially. Since this does not take much time normally you should
+    /// probably keep the default.
+    pub max_settled_nodes_initial_relevance: usize,
+    /// The maximum number of settled nodes per witness search performed when updating priorities
```

Comment:
These explanations are great, thank you! I kind of want to ask for something similar for `hierarchy_depth_factor` and `edge_quotient_factor` -- would users ever want to deviate from defaults?

Reply:
Good point, I added some documentation for these as well. I also ran some measurements for varying values of `hierarchy_depth_factor`, see here: https://gist.github.com/easbar/a7980e834206b6065e848ba29e228fdd#hierarchy-depth-factor

For the abstreet maps you sent reducing `hierarchy_depth_factor` from `0.1` to `0.01` might actually reduce preparation times further (but also might yield somewhat slower queries, ~10% according to these experiments, but that could also just be noise).